releases/net471/KnownIssues/510901-Apps-targeting-.NET-4.6.1-who-using-.NET-Standard-libraries-might-be-broken-on-.NET-4.7.1.md

```diff
@@ -0,0 +1,18 @@
+## Symptoms
+
+Application that target .NET Framework 4.6.1 who use .NET Standard libraries could start seeing TypeLoadExceptions when running on .NET 4.7.1
```

Comment:
Application --> Applications
"the" before ".NET Framework"
who --> and
could start seeing TypeLoadExceptions --> may experience \<xref:System.TypeLoadException> exceptions